turbo.json

```diff
@@ -14,6 +14,10 @@
     "dev": {
       "persistent": true,
       "cache": false
+    },
+    "test": {
+      "dependsOn": ["^build"],
```

Comment:
Does it depend on build? based on your path resolution it is using the source files & not the built files